sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -28,6 +28,20 @@ import org.apache.spark.sql.execution.stat.FrequentItems
 final class DataFrameStatFunctions private[sql](df: DataFrame) {
 
   /**
+   * Computes a pair-wise frequency table of the given columns. Also known as a contingency table.
+   * The number of distinct values for each column should be less than Int.MaxValue. The first
```

Comment:
We won't be able to handle `Int.MaxValue - 1`, which is still too large. Maybe we should say the number of distinct value pairs cannot be too large.